src/main/java/me/mrCookieSlime/Slimefun/hooks/github/ContributionsConnector.java

```diff
@@ -53,13 +61,16 @@ public String getURLSuffix() {
 	private void computeContributors(JsonArray array) {
 	    for (int i = 0; i < array.size(); i++) {
 	    	JsonObject object = array.get(i).getAsJsonObject();
-	    	
+
 	    	String name = object.get("login").getAsString();
 	    	int commits = object.get("contributions").getAsInt();
 	    	String profile = object.get("html_url").getAsString();
-	    	
-	    	if (!blacklist.contains(name)) {
-	    		Contributor contributor = SlimefunPlugin.getUtilities().contributors.computeIfAbsent(name, key -> new Contributor(name, profile));
+
+	    	if (!name.toLowerCase().contains("bot") && !blacklist.contains(name)) {
```

Comment:
What if I call myself
TheBotcher3000...

You probably just should have updated the blacklist and added a check for URL-invalid characters.